### PR TITLE
warp-625 Frontpage cleanups

### PR DESCRIPTION
**Pull request by Skadefryd**
Card data cleanup handle both types the same way from external cardData.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -370,7 +379,7 @@ h2.card-title {
   left: -100px;
   right: -100px;
   bottom: 0;
-  background-image: url('./stars-background.svg');
+  background-image: url('/stars-background.svg');
```
Comment (Skadefryd) [inline docs/index.md:382]: Should maybe fix the background stars only working locally :)

File: docs/index.md [hunk 2]
```diff
@@ -11,32 +11,48 @@ import TokensCard from './src/tokens-card.svg';
...
+    },
+    {
+      title: 'Colors',
+      href: 'foundations/styling/web/background-color#quick-reference',
+      image: { component: ColorsCard, alt: 'Two abstract shapes in grey and black colors, one bigger than the other' },
+      description: 'Get quick access to predefined color palettes for backgrounds, text, and border',
+    },
+    {
+      title: 'Tokens',
+      href: 'foundations/tokens',
```
Comment (BalbinaK) [inline docs/index.md:36]:
```suggestion
      href: 'foundations/tokens/intro/',
```
As per fix from https://github.com/warp-ds/warp-portal-poc/pull/27/files 😊

File: docs/index.md [hunk 3]
```diff
@@ -11,32 +11,48 @@ import TokensCard from './src/tokens-card.svg';
...
+      description: 'Explore how to use pre-defined utility-first CSS classes',
+    },
+    {
+      title: 'Icons',
+      href: 'components/icons',
+      image: { component: IconsCard, alt: 'Three talk bubbles that are smiling and have closed eyes' },
+      description: 'Browse our library of UI icons',
+    },
+    {
+      title: 'Colors',
```
Comment (BalbinaK) [inline docs/index.md:29]: This card links to background colors for web only. Should we have it as title then, or at least say `Colors for web`?

File: docs/index.md [hunk 4]
```diff
@@ -11,32 +11,48 @@ import TokensCard from './src/tokens-card.svg';
...
+      href: 'foundations/styling/web/background-color#quick-reference',
```
Comment (BalbinaK) [inline docs/index.md:30]: Not part of this PR, but I see that this url no longer reflects the side menu structure (it's called "CSS Classes" now) - just so it's not confusing, maybe we could apply the updated structure before/upon moving the docs to the final repository?

**Review by BalbinaK — COMMENTED** [review]
Great stuff! Just a few comments. Also, I see that `VPHero` class has bottom padding of 80px + sections with `cards` class have margin-top 40px, which makes those pretty helpful cards pretty far down on the screen. It's no show-stopper, but wondering if it's worth looking into :) 
Screenshot 2024-10-21 at 08 25 28

File: docs/index.md (hunk 3 above)
Comment (Skadefryd) [reply docs/index.md:29]: not in the scope of this pr , see other comment.

**Review by BalbinaK — APPROVED** [review]
We can take up the topics mentioned above with the designers, as it's out of this PR's scope.

**Pull request merged**